docs/tutorials/mlflow.ipynb

```diff
@@ -0,0 +1,774 @@
+{
```

Comment:
<p>There are 4 inputs and 3 outputs, are you cheating?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mlrun/mlrun/pull/4694/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>

Reply:
<p>actually i didn't notice it, good catch</p>

Reply:
<p>so there are 4 data points for each example (sepal length in cm, sepal width in cm, petal length in cm, petal width in cm) and the answer is the correspondents to each of the iris types (there are 3 types of irises in this dataset) we can see here that this example best fits the first iris</p>